evap/results/views.py

```diff
@@ -156,7 +160,9 @@ def index(request):
     course_pks = list([course.pk for course in courses_and_evaluations.keys()])
 
     # annotate each course in courses with num_evaluations
-    annotated_courses = Course.objects.filter(pk__in=course_pks).annotate(num_evaluations=Count('evaluations')).order_by('pk').defer()
+    annotated_courses = (
+        Course.objects.filter(pk__in=course_pks).annotate(num_evaluations=Count('evaluations')).order_by('pk').defer()
```

Comment:
Unrelated to this PR: Isn't this `defer()` useless here?

Reply:
I guess this is supposed to defer loading of all the attributes, since only `num_evaluations` is used. whether a `defer()` actually defers all attributes (instead of none), and why the author didn't use `only('num_evaluations')` I don't know.

(There's a high chance that I'm the author but I don't remember :D )